ca/transport.go

```diff
@@ -135,6 +135,36 @@ func (c *MutableTLSCreds) LoadNewTLSConfig(newConfig *tls.Config) error {
 	return nil
 }
 
+// UpdateCAs updates the root CAs and client CAs of the existing TLS config in place
+func (c *MutableTLSCreds) UpdateCAs(rootCAs, clientCAs *x509.CertPool) {
+	c.Lock()
+	defer c.Unlock()
+	// TODO(cyli): on upgrade to go 1.8.x, use c.config.Clone() instead
```

Comment:
Not at all - as I've said before, my machine is haunted :)  And go vet still produces errors on my machine that no one else seems to be able to replicate so I totally would believe something is wrong with my setup.

And sure, will do.